utils/build/webpack.base.js

```diff
@@ -68,6 +71,16 @@ found at http://www.apache.org/licenses/LICENSE-2.0
           }
         ]
       },
+      {
+        test: /\.tsx?$/u,
+        exclude: /node_modules/u,
+        use: [
+          {
+            loader: 'babel-loader',
```

Comment:
I’m curious @austin94, why do you prefer to use `babel-loader` (with `@babel/preset-typescript`) instead of `ts-loader` (or `ts-loader` + `babel-loader` without `@babel/preset-typescript`)? What would you lose?

And a related question: have you tried to emit declaration files at the same time as when compiling TS?

Glad to see this PR by the way 😃.

Reply:
> why do you prefer to use babel-loader (with @babel/preset-typescript) instead of ts-loader (or ts-loader + babel-loader without @babel/preset-typescript)?

This matches the default webpack config that exists in `create-react-app` now.

> have you tried to emit declaration files at the same time as when compiling TS?

This got me thinking about the webpack ordering and I've reconsidered the `ts-loader` vs `@babel/preset-typscript` decision.

I've now updated the build to use `ts-loader -> babel-loader` and emit those declarations during the build step. This should increase the build speed over time as we move more packages over.